packages/orbit-components/src/AirportIllustration/AirportIllustration.stories.tsx

```diff
@@ -36,14 +36,15 @@ Playground.story = {
   },
 };
 
-export const ListOfAllAirportIllustrations = () => {
+export const AirportIllustrationVisualTest = () => {
   return <IllustrationPrimitiveList nameOfComponent="AirportIllustration" images={NAMES} />;
 };
 
-ListOfAllAirportIllustrations.story = {
+AirportIllustrationVisualTest.story = {
   name: "List of all AirportIllustrations",
 
   parameters: {
     info: "Explore our new set of Airportillustrations for Kiwi.com.",
+    chromatic: { disableSnapshot: false },
```

Comment:
I'd add a mode here so it only triggers snapshot for largeDesktop. No need to do it for all viewports

Reply:
Agreed IRL that I'd do it